content/recipes/nestia.md

```diff
@@ -0,0 +1,698 @@
+     * 
+     * ```typescript
+     * import ts from "typescript";
+     * 
+     * const tsconfig: ts.TsConfig;
+     * const nestiaConfig: IConfiguration;
+     * 
+     * const compilerOptions: ts.CompilerOptions = {
+     *     ...tsconfig.compilerOptions,
+     *     ...(nestiaConfig.compilerOptions || {})
+     * }
+     * ```
+     */
+    compilerOptions?: ts.CompilerOptions;
+
+    /**
+     * Whether to assert parameter types or not.
+     * 
+     * If you configure this property to be `true`, all of the function parameters would be
+     * checked through the [typescript-is](https://github.com/woutervh-/typescript-is). This
+     * option would make your SDK library slower, but would be much safer in the type level
+     * even in the runtime environment.
+     */
+    assert?: boolean;
+
+    /**
+     * Whether to optimize JSON string conversion 2x faster or not.
+     * 
+     * If you configure this property to be `true`, the SDK library would utilize the
+     * [typescript-json](https://github.com/samchon/typescript-json) and the JSON string
+     * conversion speed really be 2x faster.
+     */
+    json?: boolean;
+
+    /**
+     * Building `swagger.json` is also possible.
+     * 
+     * If not specified, you can't build the `swagger.json`.
+     */
+    swagger?: IConfiguration.ISwagger;
+}
+export namespace IConfiguration
+{
+    /**
+     * List of files or directories to include or exclude to specifying the NestJS 
+     * controllers.
+     */
+    export interface IInput
+    {
+        /**
+         * List of files or directories containing the NestJS controller classes.
+         */
+        include: string[];
+
+        /**
+         * List of files or directories to be excluded.
+         */
+        exclude?: string[];
+    }
+
+    /**
+     * Building `swagger.json` is also possible.
+     */
+    export interface ISwagger
+    {
+        /**
+         * Output path of the `swagger.json`.
+         * 
+         * If you've configured only directory, the file name would be the `swagger.json`. 
+         * Otherwise you've configured the full path with file name and extension, the 
+         * `swagger.json` file would be renamed to it.
+         */
+        output: string;
+    }
+}
+```
+
+When you've completed to reading the `IConfiguration` interface, let's make the `nestia.config.ts` file directly. At first, move to your project directory. Note that the project directory means the top level directory of your project, where configuration files like `package.json` or `tsconfig.json` are placed in. After the movement, make the new `nestia` configuration file and write some script like below:
+
+```typescript
+@@filename(nestia.config)
+import type { IConfiguration } from "nestia";
+
+export const NESTIA_CONFIG: IConfiguration = {
+    input: "./src/controllers",
+    output: "./src/api",
+    json: true,
+    swagger: {
+        output: "./public/swagger.json"
+    }
+};
+export default NESTIA_CONFIG;
+```
```

Comment:
```suggestion
Please check the [nestia docs](https://github.com/samchon/nestia/wiki/Configuration) for information on the configuration file options.
```

The page is getting to be pretty long, so linking over to your documentation here would be a good idea.